packages/experimental/ActivityIndicator/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './ActivityIndicator';
```

Comment:
nit: Should probably add newline for end here and in ActivityIndicator.tsx + tsconfig.json .

I'm not sure if/why it's better but I think it's the preferred style.